apps/web/package.json

```diff
@@ -95,11 +95,12 @@
     "webpack": "4.46.0"
   },
   "scripts": {
-    "start": "PORT=4200 react-app-rewired start",
+    "start": "chmod +x ./env.sh && ./env.sh && mv env-config.js ./public/env-config.js && PORT=4200 react-app-rewired start",
     "build": "react-app-rewired build",
     "test": "react-app-rewired test",
     "precommit": "lint-staged",
-    "start:static:build": "http-server build -p 4200 --proxy http://localhost:4200?",
+    "envsetup": "chmod +x ./env.sh && ./env.sh && mv env-config.js ./public/env-config.js",
```

Comment:
I'm a bit concerned with windows users here, there is no problem that this will run inside the Dockerfile in which control the OS. But from the looks of it, windows users won't be able to start the application locally. 

Is there another way to handle this locally?

Reply:
I guess it is possible to create an env-config.js in public folder and use it for windows users...?

Reply:
I'm thinking, if we can skip this altogether and only rely on fallback values in the config.ts file you created. Because today, you can run the project without process.env files altogether.

In this way we can only run envsetup in docker and in github actions where it's needed. And locally both MAC and Windows users won't need to run this command.

So I think, you are correct with created a default file there, and we can just unify the mac and windows experience.

Reply:
Hm having hard time to follow you in your comment... 
So envsetup for docker and github actions should be easy :)
So we should add a default env-config.js file?

Reply:
Yes just like you suggested. 

And locally never run the envsetup command. So that way mac and windows users will have the same experience.

Reply:
hm I guess thats fine because those envs are public anyway...